astro/src/content/blog/custom-scopes-in-third-party-applications.mdx

```diff
@@ -0,0 +1,96 @@
+---
+publish_date: 2024-04-24
+title: "Custom Scopes, Third Party Applications, And FusionAuth"
+description: 'How to use custom scopes in a third party application with FusionAuth.'
+authors: Mark Robustelli
+image: /img/blogs/custom-scopes-in-third-party-applications/custom-spcopes-blog-header.png
+categories: Tutorial
+tags: scopes, oauth
+excerpt_separator: "{/* more */}"
+---
+
+import Aside from '/src/components/Aside.astro';
+import {RemoteCode} from '@fusionauth/astro-components';
+import LoginArchitecture from '/src/diagrams/blog/custom-scopes-in-third-party-applications/login-diagram.astro';
+
+What are third party applications and custom scopes? How would I implement them using FusionAuth?
+
+{/* more */}
+
+In today's world of API enabled applications, it is very common to open up the APIs of your application to other applications. Not just open the APIs to other applications that you and your team have created but open the APIs to applications created by other entities. These applications that consume your APIs are known as third party applications. 
+
+You will want to ensure the API calls to your application are secured. This is where scopes come into play. Similar to roles for a user, custom scopes can be created for applications. This ensures your APIs are only being called by applications you grant access to.
+
+## The Setup
+For the purposes of this blog, you will be reading about a fictional up-and-coming application named [Budget Buddy](https://github.com/FusionAuth/fusionauth-example-ruby-on-rails-custom-scopes) and the APIs created for [Change Bank](https://www.youtube.com/watch?v=CXDxNCzUspM). Change Bank is the fictional company used in many FusionAuth quickstarts. It is important to note that Change Bank uses FusionAuth as its identity provider and Budget Buddy uses a separate system.
+
+Change Bank has [existing APIs](https://github.com/FusionAuth/fusionauth-quickstart-ruby-on-rails-api) for their application. They provide methods for making change and alerting the authorities of an incident should it occur. The owner of the up-and-coming Budget Buddy application has reached out to the owner of Change Bank. He has asked if it would be possible to integrate with Change Bank by allowing Budget Buddy users to see their Change Bank balance in the Budget Buddy application. They agree and now Budget Buddy has become a third party application for Change Bank.
```

Comment:
```suggestion
Change Bank has [existing APIs](https://github.com/FusionAuth/fusionauth-quickstart-ruby-on-rails-api) for their application. They provide methods for making change and alerting the authorities of an incident should it occur. The owner of the up-and-coming Budget Buddy application has reached out to the owner of Change Bank. He has asked if it would be possible to integrate with Change Bank by allowing Budget Buddy users to see their Change Bank balance in the Budget Buddy application. They agreed and now Budget Buddy has become a third-party application for Change Bank.
```